client/src/components/NewReviewForm.js

```diff
@@ -75,7 +76,29 @@ const NewReviewForm = (props) => {
       <h5>{user.email}</h5>
       <label>
         Enter your review:
-        <input type="text" name="content" onChange={handleInputChange} value={newReview.content} />
+        <input
+          className="review-content"
+          type="text"
+          name="content"
+          onChange={handleInputChange}
+          value={newReview.content}
+        />
+      </label>
+
+      <label>
+        Enter your rating:
+        <div>
+          <select name="rating" onChange={handleInputChange} className="ratings">
+            <option value="" />
+            {[...Array(10)]
+              .map((_, i) => i + 1)
+              .map((i) => (
+                <option key={i} value={i}>
+                  {i}
+                </option>
+              ))}
+          </select>
```

Comment:
instead of a dropdown, consider an `input` field with a `type` of `number`, where you can set the min and max. dropdowns are a bit clunky for numbers, better for text options